mobile-widgets/uploadDiveLogsDE.cpp

```diff
@@ -301,10 +305,14 @@ void uploadDiveLogsDE::uploadTimeout()
 	QString err(tr("divelogs.de not responding"));
 	report_error(err.toUtf8());
 	emit uploadFinish(false, err);
+	timeout.stop();
```

Comment:
you add this after every `emit uploadFinish()` - wouldn't it make more sense to have the `timeout.stop()` in that?
Or does that not work for other reasons?

Reply:
uploadFinish() is a signal and to not have an implementation, therefore I need to stop the timeout here.

Reply:
Ahhh, I had the feeling that I was missing something - and the name confusion with the slot made it worse for me as I read the changes. Thanks for clearing this up.